openlibrary/templates/type/edition-history/view.html

```diff
@@ -9,7 +9,7 @@
 $var title: $title
 
 $ lccn = page["lccn"] and page["lccn"][0] and ("http://lccn.loc.gov/" + page["lccn"][0])
```

Comment:
This too should use https. Although loc will redirect anyhow, it's both an extra bounce and a vulnerability.